apps/aeutil/lib/bits.ex

```diff
@@ -1,14 +1,20 @@
 # Taken from http://minhajuddin.com/2016/11/01/how-to-extract-bits-from-a-binary-in-elixir/
 # License: CC BY-SA 3.0
 defmodule Aeutil.Bits do
-  def bech32_encode(prefix, bin) do
-    SegwitAddr.encode(prefix, 0, :binary.bin_to_list(bin))
+  @prefix_list ["ak$", "bh$", "bs$", "tx$", "bx$", "ok$", "cs$", "tr$"]
+
+  def encode58(prefix, data) when is_binary(data) do
```

Comment:
can you make the prefix be passed as atom here `:account_hash` , `:block_hash` ...
based on this choose the prefix

Reply:
rename to `base58c_encode`

Reply:
what about renaming this?